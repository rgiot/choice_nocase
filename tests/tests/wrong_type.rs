use choice_nocase::choice_nocase;


fn standard_case() {
	let value = 2;
	match value {
		case_choice!(2) => println!("good"),
		_ => println!("bad")
	};
}

fn main() {
	standard_case()
}