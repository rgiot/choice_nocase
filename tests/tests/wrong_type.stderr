error: cannot find macro `case_choice` in this scope
 --> tests/tests/wrong_type.rs:7:3
  |
7 |         case_choice!(2) => println!("good"),
  |         ^^^^^^^^^^^

warning: unused import: `choice_nocase::choice_nocase`
 --> tests/tests/wrong_type.rs:1:5
  |
1 | use choice_nocase::choice_nocase;
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: `#[warn(unused_imports)]` on by default
